src/quick-lint-js/error.h

```diff
@@ -278,6 +278,8 @@ class error_reporter {
   static void write_fatal_error_unimplemented_token(
       const char *qljs_file_name, int qljs_line, const char *qljs_function_name,
       token_type, const char8 *token_begin, const locator *, std::ostream &);
+
+  static bool error;
```

Comment:
Blocking: This variable belongs in `error_decorator`, not here in `error_reporter`.

Almost-blocking: Also, this variable should not be global. It should be tied to a particular `error_decorator` instance.

Reply:
I put this variable as a member in `error_decorator`.